packages/ui/docs/migration/Default.stories.mdx

```diff
@@ -0,0 +1,8 @@
+import { Meta } from '@storybook/addon-docs';
+import Link from "@cloudscape-design/components/link";
+
+<Meta title="Migration/MigratingFromLegacy" />
+
+# Migrating from NorthStar legacy
+
+Use <Link href='https://cloudscape.design/components' target='_blank'>Cloudscape components </Link>
```

Comment:
Shoudl add a note saying that detailed migration steps will be provided in seperate stories.

Reply:
Thanks. All the documentations related will be added in another PR.